azure-pipelines.yml

```diff
@@ -472,6 +472,10 @@ jobs:
     inputs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)/nuget'
       ArtifactName: nuget
+  - task: CodeSign@1
+    displayName: Validate Signatures nuget
```

Comment:
do we want to add ```continueOnError: true``` here ? I saw it is not failing, is it by default true ? just making sure.

Reply:
A lot of this checks have been changed to not fail on error but instead put logs. This is why I added the guardian break task at the end. I can give this flag a try.

Reply:
Tried this glad now and it didnt work.